java/arcs/android/host/prod/ProdArcHostService.kt

```diff
@@ -37,26 +37,26 @@ abstract class ProdArcHostService : ArcHostService() {
   class ProdAndroidHost(
     context: Context,
     lifecycle: Lifecycle,
-    coroutineContext: CoroutineContext,
-    arcSerializationCoroutineContext: CoroutineContext,
+    auxiliaryScope: CoroutineScope,
+    arcSerializationScope: CoroutineScope,
```

Comment:
We should not pass two different scopes here. We only want one parent scope for a component.

Reply:
I think the goal here is to provide a way to specify a different dispatcher for arc serialization. This can be done by providing a `CoroutineDispatcher` parameter here, and then using `scope.launch(arcSerializationDispatcher)`.